src/gen/pdb2pqr.py

```diff
@@ -0,0 +1,116 @@
+
+#easy way to get pdb2pqr on CLI
+import time
+from beaupy import confirm, prompt, select, select_multiple
+#from beaupy.spinners import *
+from rich.console import Console
+
+console = Console()
+#making the cl prompt
+result = ['pdb2pqr']
+
+
+#pKa Options
+if confirm("Would you like to use PROPKA to assign protonation states at provided pH?"):
+    #get pH from user
+    pH = prompt('pH value?', target_type=float, validator=lambda count: count > 0 and count < 14)
+    #making the cl prompt
+    pH_str = f"--titration-state-method=propka --with-ph={pH}"
+    result.append(pH_str)
+
+
+#forcefield options
+forcefields = [
+        "AMBER",
+        "SCHARM",
+        "PEOEPB",
+        "PARSE",
+        "SWANSON",
+        "TYLO6",
+        "[red]User-defined Forcefield[/red]"
+    ]
+console.print("Please choose a forcefield to use:")
+# Choose one item from a list
+forcefield = select(forcefields, cursor="🢧", cursor_style="cyan")
+#making the cl prompt
+if forcefield != "[red]User-defined Forcefield[/red]":
+    forcefield_str = f"--ff={forcefield}"
+    result.append(forcefield_str)
+    
+else:
+    pass #still have to do something
```

Comment:
This is fine for now, but look into this with joos and see how pdb2pqr handles it.

Reply:
You can do it later